src/index.html

```diff
@@ -0,0 +1,113 @@
+<!DOCTYPE html>
+<html lang="ru">
+
+<head>
+  <meta charset="UTF-8">
+  <meta name="viewport" content="width=device-width, initial-scale=1.0">
+  <title>Auto-Magazine</title>
+</head>
+
+<body class="page">
+  <header class="menu">
+    <nav class="menu__nav">
+      <ul class="menu__nav-list">
+        <li class="menu__nav-item"><a class="menu__nav-link" href="#">ФАКТЫ</a></li>
+        <li class="menu__nav-item"><a class="menu__nav-link" href="#">ГАЛЕРЕЯ</a></li>
+        <li class="menu__nav-item"><a class="menu__nav-link" href="#">ПРЕССА</a></li>
+      </ul>
+    </nav>
+    <a href="#" class="logo logo_type_header"></a>
+    <button class="burger active-el" type="button" aria-pressed="false"></button>
+  </header>
+  <main class="сontent">
+    <section class="slide">
+      <p class="slide__brand">volkswagen T1</p>
+      <h1 class="slide__title">Новая версия популярного микроавтобуса</h1>
+      <h2 class="slide__subtitle">С ЭЛЕКТРО ДВИГАТЕЛЕМ</h2>
+      <button class="slide__button" type="button">Подробнее</button>
+      <img class="slide__photo" src="../images/bus.png" alt="Фото автобуса">
+    </section>
+    <section class="pros-cons">
+      <div class="pros-cons__description">
+        <p class="pros-cons__text"><span class="pros-cons__text-bold">VW T1 Electro:</span> Отделение Volkswagen
+          Commercial Vehicles представило концепт
+          e&#8209;Bulli: классический вэн T1 1966 года
+          с&nbsp;электрической силовой установкой.
+          По&nbsp;словам производителя, проект получит коммерческое продолжение, и через фирму&#8209;партнера eClassics
+          наборы для
+          конверсии микроавтобусов в электрокары
+          и&nbsp;уже готовые машины будут продавать частным клиентам.
+        </p>
+      </div>
+      <div class="pros-cons__container">
+        <div class="pros-cons__wrapper pros-cons__advantages">
+          <h2 class="pros-cons__title pros-cons__advantages-title">ПЛЮСЫ</h2>
+          <ul class="pros-cons__list pros-cons__advantages-list">
+            <li class="pros-cons__item pros-cons__advantages-item">+ Экологичный</li>
+            <li class="pros-cons__item pros-cons__advantages-item">+ Экономный</li>
+            <li class="pros-cons__item pros-cons__advantages-item">+ Тихий</li>
+          </ul>
+        </div>
+        <div class="pros-cons__wrapper pros-cons__disadvantages">
+          <h2 class="pros-cons__title pros-cons__disadvantages-title">МИНУСЫ</h2>
+          <ul class="pros-cons__list pros-cons__disadvantages-list">
+            <li class="pros-cons__item pros-cons__disadvantages-item">- Дорогой</li>
+            <li class="pros-cons__item pros-cons__disadvantages-item">- Подзарядка</li>
+          </ul>
+        </div>
```

Comment:
По семантике тут лучше использовать [список описаний](https://developer.mozilla.org/ru/docs/Web/HTML/Element/dl). Будет что-то вроде:
```suggestion
      <dl class="pros-cons__container">
        <div class="pros-cons__group pros-cons__advantages">
          <dt class="pros-cons__title pros-cons__advantages-title">ПЛЮСЫ</dt>
          <dd class="pros-cons__item pros-cons__advantages-item">+ Экологичный</dd>
          <dd class="pros-cons__item pros-cons__advantages-item">+ Экономный</dd>
          <dd class="pros-cons__item pros-cons__advantages-item">+ Тихий</dd>
        </div>
        <div class="pros-cons__group pros-cons__disadvantages">
          <dt class="pros-cons__title pros-cons__disadvantages-title">МИНУСЫ</dt>
          <dd class="pros-cons__item pros-cons__disadvantages-item">- Дорогой</dd>
          <dd class="pros-cons__item pros-cons__disadvantages-item">- Подзарядка</dd>
        </div>
      </dl>
```